definitions/config.lua

```diff
@@ -13,6 +13,8 @@ return {
     'command',
     'operator',
     'meta_command',
-  }
-}
+  },
 
+  -- create you custom name prefix
```

Comment:
this is a user file and a user would not understand this at first glance, a more descriptive variable name may help